src/modules/Functions.js

```diff
@@ -96,7 +106,18 @@ function DisplayCards(data) {
       </div>
           `;
   });
-//   AddComentEventListener();
 }
 
-export { DisplayPopup, DisplayCards };
+function Counter(data) {
+//   const LikesCounter = document.querySelectorAll('.likes-counter');
+  ProductsCounter.innerHTML = data.products.length;
+  // data.likes.forEach(item => {
+  //     LikesCounter.forEach(counter=>{
+  //         if(item.item_id === counter.id) {
+  //             counter.innerHTML=item.likes
+  //         }
+  //     })
+  // })
+}
```

Comment:
Kindly, remove the comments to make your work much cleaner. #27

Reply:
Done